crt_portal/cts_forms/templates/forms/report_base.html

```diff
@@ -65,17 +65,27 @@ <h1 class="padding-top-5 margin-0">
         {{ hidden_field }}
       {% endfor %}
 
-      <h2 class="margin-top-5 margin-bottom-4">
-        {{ current_step_name }}
+      <h2 class="margin-top-5 margin-bottom-0">
+        {{ current_step_title }}
```

Comment:
This variable captures the more verbose page title, e.g. "Please provide details on what happened" instead of "Details."